README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
I agree with you @n1k0 . I'm currently using minItems in my application b/c that's the `json-schema` way of doing it. If we added documentation describing it, then it'll be more obvious that it's the way to do it.